templates/puppet.conf.erb

```diff
@@ -64,3 +64,6 @@
 <% scope.lookupvar("puppet::additional_settings").sort_by {|k, v| k}.each do |key, value| -%>
     <%= key %> = <%= value %>
 <% end -%>
+<% if @stringify_facts == false -%>
```

Comment:
The puppet internal default is currently true (see https://docs.puppetlabs.com/references/latest/configuration.html#stringifyfacts). I thought if the parameter doesn't deviate from the default (set to true or undef) i don't set it in the config at all.